src/main/java/com/powsybl/openloadflow/sensi/WoodburyEngine.java

```diff
@@ -84,12 +84,12 @@ private void setAlphas(DenseMatrix states, int columnState) {
 
     /**
      * Calculate post-contingency states values using pre-contingency states values and some flow transfer factors (alphas).
-     * @return a matrix of post-contingency voltage angle states.
+     * @return a matrix of post-contingency voltage angle states (which is the same modified pre-contingency states java object).
      */
     public DenseMatrix run(DenseMatrix preContingencyStates) {
         Objects.requireNonNull(preContingencyStates);
```

Comment:
The API xould be clearer with something like
public void toPostContingencyStates(DenseMatrix preContingencyStates);

It would clarify that the input matrix is modified.